string axis
int32 scale

